spec/lib/api/login-spec.js

```diff
@@ -54,10 +54,12 @@ describe('Http.Api.Login', function () {
         return [
             dihelper.simpleWrapper(require('swagger-express-mw'), 'swagger'),
             dihelper.simpleWrapper(ws.Server, 'WebSocketServer'),
+            dihelper.simpleWrapper({}, 'TaskGraph.TaskGraph'),
+            dihelper.simpleWrapper({}, 'TaskGraph.Store'),
             dihelper.simpleWrapper({}, 'Task.Services.OBM'),
             dihelper.simpleWrapper({}, 'ipmi-obm-service'),
-            dihelper.requireWrapper('rimraf', 'rimraf'),
-            dihelper.requireWrapper('os-tmpdir', 'osTmpdir'),
+            dihelper.requireWrapper('rimraf', 'rimraf', undefined, __dirname),
```

Comment:
Okay - call me clueless - why the additional `undefined` and `__dirname` passed into the DI system here?